frontend/src/main/java/starter/project/frontend/CarController.java

```diff
@@ -34,16 +32,39 @@ public Car getCar(@PathVariable("carId") int carId) {
     @GetMapping(path = "/cars/{list}")
     public List<Car> getBatchCars(@PathVariable List<String> list) {
         List<Car> newList = new ArrayList<>();
-        for(String entry : list) {
-         newList.add(map.get(entry));
+        for (String entry : list) {
+            newList.add(map.get(entry));
         }
         return newList;
     }
+
     @PostMapping(path = "/cars", consumes = {"application/json"})
     public Map<String, Car> createCarList(@RequestBody List<Car> carList) {
         carNewList = carList;
```

Comment:
Should probably rename carNewList as it's the actual list not a new one. It's kind of confusing.

Reply:
Especially as there's several newlIst names around

Reply:
resolved